docs/requirements.txt

```diff
@@ -0,0 +1,3 @@
+six
```

Comment:
I'll play with it, but the first try installed all the inspire-next deps too (flask and such)